helpers/calendar/ratios_and_utilization.py

```diff
@@ -26,22 +26,25 @@ def get_analytics_ratios(self, query, start, end):
             self, query)
 
         checkins = 0
-        events_list = []
         cancellations = 0
+        calendar_events_list = []
+        app_events_list = []
 
         for room in rooms:
-            checkins, cancellations, events_list = RoomAnalyticsRatios.retrieve_cancellations_and_checkins_for_room(self,  # noqa
+            checkins, cancellations, calendar_events_list, app_events_list = RoomAnalyticsRatios.retrieve_cancellations_and_checkins_for_room(self,  # noqa
```

Comment:
Try to avoid using noqa. Also instead of passing in self i believe you should create an object of that class before making the method call.

Reply:
I do not mean the method implementation but rather the method call.